falcon/response.py

```diff
@@ -746,6 +749,10 @@ def add_link(self, target, rel, title=None, title_star=None,
         if anchor is not None:
             value += '; anchor="' + uri_encode(anchor) + '"'
 
+        if crossorigin is not None:
+            if crossorigin in ['anonymous', 'use-credentials']:
```

Comment:
Since setting crossorigin to a blank value is the same as `anonymous', we can check for that case and avoid performing the extra concatenation, e.g.:

```py
value += '; crossorigin'
```